class exited_game(Exception):
    def __init__(self):
        pass
